### PR TITLE
✨Add Visitor Counter feature done ! 

### PR DESCRIPTION
**Pull request by Subhajit-2023-44**
Hello, PR
Add Visitor Counter feature done properly - #643 #649

Screenshot 2024-10-24 163237

Before -
Screenshot 2024-10-24 163244

## Type of PR
- **Add X in the box to specify the improvement type.**
- [ ] Bug fix
- [x] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): ___________  

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by Subhajit-2023-44** [comment]
@swaraj-das @AnitSarkar123 done, merge my PR with level 3,

Thank You

**Comment by Subhajit-2023-44** [comment]
@swaraj-das @AnitSarkar123

File: index.html [hunk 1]
```diff
@@ -654,6 +654,18 @@ <h2>Feedback Form</h2>
...
+    <link rel="stylesheet" href="visi.css">
+    <!-- ############### Footer ############### -->
+    <div class="visitor-counter">
+      <div>Visitor</div>
+      <div class="website-counter"></div>
+    </div>
+    <script src="path/to/visitorCounter.js"></script>
+    <script src="./assets/css/visitors.css"></script>
+
+      <script src="visi.js"></script>
```
Comment (swaraj-das) [inline index.html:667]: Please don’t add separate files named `visi.css` and `visi.js` for this feature. Instead, integrate the code directly into `style.css` and `script.js`. This approach will help keep the file structure simple and prevent confusion, ensuring that other developers can easily understand and navigate the project.

File: index.html [hunk 2]
```diff
@@ -654,6 +654,18 @@ <h2>Feedback Form</h2>
...
             </div>
         </div>
+        <br /><br />
+    <link rel="stylesheet" href="visi.css">
+    <!-- ############### Footer ############### -->
+    <div class="visitor-counter">
+      <div>Visitor</div>
+      <div class="website-counter"></div>
+    </div>
+    <script src="path/to/visitorCounter.js"></script>
```
Comment (swaraj-das) [inline index.html:664]: <script src="path/to/visitorCounter.js"></script>
<script src="./assets/css/visitors.css"></script>
What is the meaning of these two lines in code? There is not any path.

File: index.html (hunk 1 above)
Comment (Subhajit-2023-44) [reply index.html:667]: okay 😭😭 I will give new PR wait

**Pull request closed without merging**